packages/react/src/themes/base/components/Icon/iconVariables.ts

```diff
@@ -1,19 +1,34 @@
 import { pxToRem } from '../../../../lib'
 
-export type IconSizeModifier = 'x' | 'xx'
-
 export interface IconVariables {
-  [key: string]: object | string | number | boolean | undefined
-  color?: string
-  borderColor?: string
-
+  color: string
+  borderColor: string
+  backgroundColor: string
+  disabledColor: string
   horizontalSpace: string
+
+  smallestSize: string
+  smallerSize: string
+  smallSize: string
+  mediumSize: string
+  largeSize: string
+  largerSize: string
+  largestSize: string
```

Comment:
Size values where moved to variables

Reply:
Not sure that these size values are general enough for base theme...

Reply:
Base theme should support `size` prop properly and we should avoid nesting in variables. Do you have better proposal? 🐱

Reply:
I agree that we should have variables and use this, I was saying about the size values (which now reflect the Teams theme) If you think these values are general and should be defined as this, then I am okay